docs/dev/libs/gelly/library_methods.md

```diff
@@ -242,6 +242,28 @@ The algorithm takes a directed, vertex (and possibly edge) attributed graph as i
 vertex represents a group of vertices and each edge represents a group of edges from the input graph. Furthermore, each
 vertex and edge in the output graph stores the common group value and the number of represented elements.
 
+## Minimum Spanning Tree
+
+#### Overview
+This is an implementation of the distributed minimum spanning tree (MST) algorithm. A minimum spanning tree for a connected and
+undirected graph is a subset of edges with minimum possible total edge weight that connects all the vertices without cycles. 
+One of the possible use cases for MST could be laying out cables that connect all buildings with the minimum total cable length.
+
+#### Details
+Unlike a sequential version of the algorithm, a distributed MST algorithm is based on the message-passing model. 
+We use [vertex-centric iterations](#vertex-centric-iterations) to implement the Borůvka algorithm described in 
+[this paper](http://ieeexplore.ieee.org/abstract/document/508073/). As there are different steps inside the iteration,
```

Comment:
Yes greg, you are right. A pdf is surely better than a web site. Think it over, I want to change the paper to https://cs.uwaterloo.ca/~kdaudjee/comparison.pdf since it fits more.